controllers/openstack_ansibleee_controller.go

```diff
@@ -75,6 +75,22 @@ func (r *OpenStackAnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return ctrl.Result{}, err
 	}
 
+	// Initialize Status
+	if instance.Status.Conditions == nil {
+		instance.Status.Conditions = condition.Conditions{}
+
+		cl := condition.CreateList(
+			condition.UnknownCondition(redhatcomv1alpha1.AnsibleExecutionJobReadyCondition, condition.InitReason, redhatcomv1alpha1.AnsibleExecutionJobInitMessage),
+		)
+
+		instance.Status.Conditions.Init(&cl)
+		instance.Status.JobCompleted = false
```

Comment:
nit: either we don't need the default kubebuilder definition in the struct or if that is effective then we don't need to initialize the field here explicitly.